doc/chapters/api/index.rst

```diff
@@ -7,105 +6,291 @@ API
 This chapter documents the DBus and C API for interacting with the SoftwareContainerAgent. This is
```

Comment:
Since you have used "D-Bus" further down in the document, I think you should also change this instance.